.github/workflows/release-builder.yml

```
@@ -10,6 +10,8 @@ on:
     # Runs every Friday at 12:00 PM IST (6:30 AM UTC)
     - cron: '30 6 * * 5'
   workflow_dispatch:
+    branches:
```

actionlint (1.7.12)

[error] expected "inputs" key for "workflow_dispatch" section but got "branches"

(syntax-check)